src/api/json/catalog.json

```diff
@@ -26,6 +26,18 @@
       ],
       "url": "https://json.schemastore.org/aiproj.json"
     },
+    {
+      "name": "Airlock Microgateway",
+      "description": "Airlock Microgateway configuration schema",
+      "url": "https://json.schemastore.org/airlock-microgateway-3.0.json",
+      "fileMatch": [
+        "config.yaml",
```

Comment:
This is too generic for file matching. Many YAML files are called `config.yaml` that aren't an Airlock config. Please either rename to `airlock-config.yaml` or something else less generic, or remove the fileMatch.

Reply:
Makes sense! I changed the file match pattern.